src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -135,21 +135,14 @@ void EnGinkoMan_Idle(EnGinkoMan* this, GlobalContext* globalCtx) {
     }
 }
 
-#if NON_EQUIVELENT
-// non-equiv: case 14 is possibly still non-equiv but def does not match, hard to tell
-// if else inside of jump tables makes mips2c confused, also late rodata
 // action func: non-input dialogue
 void EnGinkoMan_DepositDialogue(EnGinkoMan* this, GlobalContext* globalCtx) { // 80418E34
-    u32 tempEDC;
-    s16* previousBankValue;
-
     if (func_80147624(globalCtx) == 0) {
         return;
     }
 
-    switch (this->curTextId - 0x44C) {
-
-        case 0: // "Hey there, little guy!  Won't you deposit some Rupees? (first dialogue)
+    switch (this->curTextId) {
+        case 0x44C: // "Hey there, little guy!  Won't you deposit some Rupees? (first dialogue)
```

Comment:
I don't know why this never occurred to me, it was hinted by the switches being different in different functions, but I didn't think to try it.

Changing all of the cases back to full value must have sucked...

Reply:
It actually matched without changing the switches back to their full value. Switching it back was more of a post-match documentation step since mips nearly always shifts switches down to 0 for Jump Table conventions